requirements/dev.txt

```diff
@@ -18,7 +18,7 @@ contextlib2==0.6.0.post1  # via importlib-metadata
 diff-cover==2.4.0
 distlib==0.3.0            # via caniusepython3
 django-config-models==1.0.1
-django==1.11.25
+django==1.11.26 ; python_version < "3"
```

Comment:
Hmm, this isn't really what we want in the output; Django won't be correctly pinned in Python 3 dev environments.  We may need to give up on the upper constraint on Django in `base.in` and just leave it as `Django>=1.11`; I don't think there's a good way to get `pip-compile` to generate what we want here with an environment marker involved (there's an [open issue](https://github.com/jazzband/pip-tools/issues/518) for this).